index.html

```diff
@@ -1,312 +1,192 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Anyxz</title>
-    <meta property="og:title" content="Ez Video Background Semantic Segmentation" />
-    <meta property="og:type" content="website" />
-    <!-- Update reqd -->
-    <meta property="og:image" content="#" />
-    <meta property="og:description" content="Ez Change/Modify segmented video background!" />
-    <!-- Update reqd -->
-    <meta property="og:url" content="#" />
-    <link rel="shortcut icon" href="./assets/favicon.png" type="image/x-icon" />
-    <link rel="stylesheet" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Alef&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/5.0.0-alpha1/css/bootstrap.min.css" />
-    <!-- Don't alter the sequence -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>
-  </head>
 
-  <body>
-    <h1 id="title">Anyxz</h1>
-    
-    <!-- Experiment Alert Start-->
-    <div class="alert alert-primary" role="alert" id="expmnt-alert">
-      This is an experiment!,
-      <a href="mailto:neilblaze007@gmail.com">send feedback</a>
-    </div>
-    <!-- Experiment Alert End -->
+<head>
+  <meta charset="UTF-8" />
+  <!-- i am changing the title to improve seo -->
+  <title>[ANYXZ] - EDITS YOUR BACKGROUND DURING LIVE VIDEO CHATS</title>
```

Comment:
Update to - "Anyxz" from `[ANYXZ] - EDITS YOUR BACKGROUND DURING LIVE VIDEO CHATS`

Reply:
should i remove only the square brackets or the whole line?

Reply:
Replace the whole line by "Anyxz"
Also, follow [this](https://ahrefs.com/blog/seo-meta-tags/) for upgradation.

Reply:
ok noted.

Reply:
ok noted.